server/webdriver_tests/place_explorer_test.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import urllib
+
+from flask import abort, url_for
+from flask_testing import LiveServerTestCase
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.action_chains import ActionChains
+import logging
+import time
+
+from main import app
+
+
+USA_URL = '/place?dcid=country/USA'
+USA_DEMOGRAPHICS_URL = '/place?dcid=country/USA&topic=Demographics'
+
+class TestBase(LiveServerTestCase):
+
+    def create_app(self):
+        return app
+
+    def setUp(self):
+        """
+        Will be called before every test
+        """
+        chrome_options = webdriver.ChromeOptions()
+        chrome_options.add_argument('--headless')
+        chrome_options.add_argument('--no-sandbox')
+        chrome_options.add_argument('--disable-dev-shm-usage')
+
+        self.driver = webdriver.Chrome(chrome_options=chrome_options)
+        self.driver.get(self.get_server_url() + USA_URL)
```

Comment:
can move this and test_page_server() to the concrete test class below. The TestBase can just have env setup without any specific test.